src/test/java/de/retest/recheck/util/ApprovalsUtil.java

```diff
@@ -153,6 +153,7 @@ public void increment() {}
 		@Override
 		public void report( final String received, final String approved ) {
 			try {
+				Paths.get( approved ).toFile().delete();
 				Files.copy( Paths.get( received ), Paths.get( approved ) );
```

Comment:
The `copy` method allows you to pass `CopyOptions` like:

```java
Files.copy( Paths.get( received ), Paths.get( approved ), StandardCopyOption.REPLACE_EXISTING );
```

Then you can remove the inserted code.